Ports/.port_include.sh

```diff
@@ -101,6 +102,17 @@ install_main_launcher() {
     fi
 }
 
+install_icon() {
+    command -v convert >/dev/null 2>&1
+    convert_exists=$?
+    if [ -n "$icon_file" ] && [ -n "$launcher_command" ] && [ "$convert_exists" == 0 ]; then
+	run convert "${icon_file}[0]" app-16x16.png
+	run convert "${icon_file}[1]" app-32x32.png
+	run objcopy --add-section serenity_icon_s="app-16x16.png" "${DESTDIR}${launcher_command}"
+	run objcopy --add-section serenity_icon_m="app-32x32.png" "${DESTDIR}${launcher_command}"
```

Comment:
How would this work with multiple launchers? (See the stpuzzles port)

Reply:
I updated the script to parameterize the icon and launcher files -- mirroring the design of `install_main_launcher` and `install_launcher`.

By default, an install will only run `install_main_icon`, but the package.sh script can call `install_icon` with whatever icon and launcher combos desired.

Thanks a ton for the review :)